conversant/prompt_chatbot.py

```diff
@@ -280,10 +374,12 @@ def get_current_prompt(self, query: str, max_context_examples: int = None) -> st
             context_prompt_lines.append(self.prompt.create_interaction_string(**turn))
         context_prompt = self.prompt.example_separator + "".join(context_prompt_lines)
 
-        # get query prompt
-        query_prompt = self.prompt.create_interaction_string(query)
+        current_prompt = base_prompt + context_prompt
 
-        current_prompt = base_prompt + context_prompt + query_prompt
+        # get query prompt
+        if query != "":
```

Comment:
Just out of curiosity why do we need to check if this is empty before concatenating the strings? I don't really have the context on how Interactions work 🙃

Reply:
Because of the way it is implemented, it throws an exception if the prompt is empty. I think it also makes sense not to update the chat history if the prompt (in this case an utterance) is empty. 🤔